assets/src/dashboard/components/navigation-bar.js

```diff
@@ -29,8 +29,24 @@ import styled from 'styled-components';
  */
 import { useRouteHistory } from '../app/router';
 import { ReactComponent as WebStoriesLogoSVG } from '../images/logo.svg';
-import { BUTTON_TYPES } from '../constants';
+import { BUTTON_TYPES, NEW_STORY_URL } from '../constants';
 import Button from './button';
+import Dropdown from './dropdown';
+
+const Link = styled.a`
+  font-family: ${({ theme }) => theme.fonts.tab.family};
+  font-size: ${({ theme }) => theme.fonts.tab.size};
+  line-height: ${({ theme }) => theme.fonts.tab.lineHeight};
+  letter-spacing: ${({ theme }) => theme.fonts.tab.letterSpacing};
+  text-decoration: none;
+  margin-left: 40px;
+  color: ${({ theme, active }) =>
+    active ? theme.colors.gray900 : theme.colors.gray500};
```

Comment:
Super nit, but I think the figma says the inactive color should be `gray600`:
![image](https://user-images.githubusercontent.com/40646372/77204380-d8f55800-6aaf-11ea-9a71-6be352976dc2.png)

Reply:
Good find!